packages/core/src/storage/migration/__tests__/UpdateAssistant.test.ts

```diff
@@ -54,17 +54,25 @@ describe('UpdateAssistant', () => {
     it('should return true when a new wallet is created', async () => {
       expect(await updateAssistant.isUpToDate()).toBe(true)
     })
+
+    it('should return true for a lower version than current storage', async () => {
+      expect(await updateAssistant.isUpToDate('0.2')).toBe(true)
+    })
+
+    it('should return true for current agent storage version', async () => {
```

Comment:
Well, we can also make it return false by sending '0.4' parameter :sweat_smile:

Also I guess `CURRENT_FRAMEWORK_STORAGE_VERSION` can be used instead of hard-coded 0.3.  Could it be possible to use this constant in update the test title? :thinking: